diff --git a/node_modules/@epfl/epfl-sti-react-library/dist/Navigations/Avatar.js b/node_modules/@epfl/epfl-sti-react-library/dist/Navigations/Avatar.js
index 5c848d2..c849702 100644
--- a/node_modules/@epfl/epfl-sti-react-library/dist/Navigations/Avatar.js
+++ b/node_modules/@epfl/epfl-sti-react-library/dist/Navigations/Avatar.js
@@ -9,7 +9,7 @@ var _react = _interopRequireDefault(require("react"));
 
 var _propTypes = _interopRequireDefault(require("prop-types"));
 
-var _sti_logo = _interopRequireDefault(require("./sti_logo.png"));
+var _sti_logo = "";
 
 require("./index.css");
 
diff --git a/node_modules/@epfl/epfl-sti-react-library/dist/Navigations/mainMenuHelpers.js b/node_modules/@epfl/epfl-sti-react-library/dist/Navigations/mainMenuHelpers.js
index 2fa6af9..8971b69 100644
--- a/node_modules/@epfl/epfl-sti-react-library/dist/Navigations/mainMenuHelpers.js
+++ b/node_modules/@epfl/epfl-sti-react-library/dist/Navigations/mainMenuHelpers.js
@@ -5,9 +5,9 @@ Object.defineProperty(exports, "__esModule", {
 });
 exports.renderWithoutParent = exports.renderWithParentWithoutMenus = exports.renderWithParentWithMenus = exports.renderWithParent = exports.findMenuDisplay = exports.findCurrentItem = void 0;
 
-var _arrowLeft = _interopRequireDefault(require("./arrow-left.svg"));
+var _arrowLeft = "";
 
-var _chevronRight = _interopRequireDefault(require("./chevron-right.svg"));
+var _chevronRight = "";
 
 function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }
 
diff --git a/node_modules/@epfl/epfl-sti-react-library/package.json b/node_modules/@epfl/epfl-sti-react-library/package.json
index 83ae8e5..f61dd04 100644
--- a/node_modules/@epfl/epfl-sti-react-library/package.json
+++ b/node_modules/@epfl/epfl-sti-react-library/package.json
@@ -46,7 +48,6 @@
     "lodash.filter": "^4.6.0",
     "react-cookie": "4.0.3",
     "react-plotly.js": "2.5.1",
-    "react-router-dom": "5.2.0",
     "react-scroll": "^1.8.6",
     "react-slick": "^0.28.1",
     "rimraf": "3.0.2",
@@ -89,6 +90,9 @@
     "standard": "^16.0.4",
     "web-vitals": "0.2.4"
   },
+  "peerDependencies": {
+    "react-router-dom": "5.2.0"
+  },
   "eslintConfig": {
     "extends": [
       "react-app",
